src/main/java/uk/gov/hmcts/reform/pbis/config/ApplicationConfig.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.pbis.config;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.hmcts.reform.pbis.model.EmailTemplateMapping;
+
+@Configuration
+@ConfigurationProperties
```

Comment:
IntelliJ complaints and [suggests adding dependency](https://docs.spring.io/spring-boot/docs/1.5.6.RELEASE/reference/html/configuration-metadata.html#configuration-metadata-annotation-processor).

Reply:
@ConfigurationProperties are in spring-boot library, which you should see in the dependency graph (`./gradlew dependencies`).

I've cleaned the cache and couldn't reproduce this problem.